bundles/org.pitest.pitclipse.core/src/org/pitest/pitclipse/core/preferences/PitPreferences.java

```diff
@@ -20,25 +20,39 @@
  * Constants used to deal with Pitclipse's preferences.
  */
 public final class PitPreferences {
+    public static final String MUTATORS_LABEL = "Mutators";
+    public static final String MUTATORS = "pitMutators";
+    
+    public static final String INDIVIDUAL_MUTATORS = "pitIndividualMutators";
 
-    public static final String PIT_MUTATORS = "pitMutators";
-
-    public static final String USE_INCREMENTAL_ANALYSIS = "Use &incremental analysis";
-    public static final String EXCLUDE_CLASSES_FROM_PIT = "E&xcluded classes (e.g.*IntTest)";
-    public static final String MUTATION_TESTS_RUN_IN_PARALLEL = "Mutation tests run in para&llel";
-    public static final String EXCLUDE_METHODS_FROM_PIT = "Excluded &methods (e.g.*toString*)";
-    public static final String AVOID_CALLS_FROM_PIT = "&Avoid calls to";
-    public static final String PIT_TIMEOUT = "Pit Ti&meout";
-    public static final String PIT_TIMEOUT_FACTOR = "Timeout &Factor";
-    public static final String EXCLUDED_METHODS = "excludedMethods";
-    public static final String AVOID_CALLS_TO = "avoidCallsTo";
-    public static final String EXCLUDED_CLASSES = "excludedClasses";
+    public static final String INCREMENTAL_ANALYSIS_LABEL = "Use &incremental analysis";
     public static final String INCREMENTAL_ANALYSIS = "incrementalAnalysis";
-    public static final String PIT_EXECUTION_MODE = "pitExecutionMode";
+
+    public static final String EXCLUDE_CLASSES_LABEL = "E&xcluded classes (e.g.*IntTest)";
+    public static final String EXCLUDED_CLASSES = "excludedClasses";
+
+    public static final String RUN_IN_PARALLEL_LABEL = "Mutation tests run in para&llel";
     public static final String RUN_IN_PARALLEL = "runInParallel";
+
+    public static final String EXCLUDE_METHODS_LABEL = "Excluded &methods (e.g.*toString*)";
+    public static final String EXCLUDE_METHODS = "excludedMethods";
+
+    public static final String AVOID_CALLS_LABEL = "&Avoid calls to";
+    public static final String AVOID_CALLS = "avoidCallsTo";
+
+    public static final String TIMEOUT_LABEL = "Pit Ti&meout";
     public static final String TIMEOUT = "pitTimeout";
+
+    public static final String TIMEOUT_FACTOR_LABEL = "Timeout &Factor";
     public static final String TIMEOUT_FACTOR = "pitTimeoutFactor";
     
+
+    public static final String EXECUTION_SCOPE_LABEL = "Pit execution scope";
+    public static final String EXECUTION_SCOPE = "pitExecutionMode";
+
+    public static final String PREFERENCE_DESCRIPTION_LABEL = "Pitclipse Preferences";
+    public static final String MUTATORS_DESCRIPTION_LABEL = "Mutator Preferences";
+
```

Comment:
You seem to have rename some constants here. What is the rationale for this change? IMHO that's out of the scope of this PR and, overall, some changes don't look justified to me. For instance, `EXCLUDED_METHODS` became `EXCLUDE_METHODS`; why? It's the key to store the name of methods that are excluded from the analysis so the previous name made sense.
 - `

Reply:
I wanted to create a clear naming scheme for all the labels and there corresponding keys.
>
>[…] For instance, EXCLUDED_METHODS became EXCLUDE_METHODS[…]

In this particular case it was an oversight, because I used the old name ‚EXCLUDE_METHODS_FROM_PIT‘.

The whole reason for this renaming started, because the tests used different static Strings and I wanted them to be reliable, even if the text itself changes.

Reply:
I don't really have a strong opinion on that, so I leave it to @echebbi

Reply:
I changed it to be `EXCLUDED...` now. I think this is the better name.

Which I found more concerning is the `EXECUTION_SCOPE...`. In the UI we are using the term scope, but the key uses the term mode.

Reply:
Yes, the UI talks about "scope" because that's what makes the more sense to the user. Maybe the constant should be renamed accordingly — I'm fully open to such changes — but I'd prefer to see them addressed in a new issue/PR. 

The reason is that this PR already brings a lot of changes, and renaming constants is not a trivial thing. I mean, sure, technically it's easy to do but:

- we have to discuss whether the changes are worth it (e.g., do the new names make more sense?),
- we have to make sure the codebase stays consistent. For instance, you renamed `PIT_EXECUTION_MODE` as `EXECUTION_SCOPE` but did you notice that we use the term "execution mode" elsewhere? We have a [`PitExecutionMode` enum](https://github.com/pitest/pitclipse/blob/bd00f0dafc1fbafb2db3df93bbef9d9adfe83de0/bundles/org.pitest.pitclipse.runner/src/org/pitest/pitclipse/runner/config/PitExecutionMode.java#L19), a [`setExecutionMode` method](https://github.com/pitest/pitclipse/blob/bd00f0dafc1fbafb2db3df93bbef9d9adfe83de0/bundles/org.pitest.pitclipse.core/src/org/pitest/pitclipse/core/PitCoreActivator.java#L304), etc. It's important to keep the same terminology throughout the project otherwise it can become very confusing. In other words, if we decide to rename a constant we may have to rename classes, fields or methods as well.

Besides, if by "key" you were refering to the value of the constant (`pitExecutionMode`) then I don't think we'll change it because it would likely break compatibility with existing run configurations.

Reply:
>
>Yes, the UI talks about "scope" because that's what makes the more sense to the user. Maybe the constant should be renamed accordingly — I'm fully open to such changes — but I'd prefer to see them addressed in a new issue/PR.

I see that now. I wasn't aware of this methodology, since this is my first big open source project I'm contributing to.
Personally, I would like to leave the changes in this PR, because they are connected with the changes. If you prefer otherwise, I understand that.

>The reason is that this PR already brings a lot of changes, and renaming constants is not a trivial thing. I mean, sure, technically it's easy to do but:
  *  we have to discuss whether the changes are worth it (e.g., do the new names make more sense?),
  * we have to make sure the codebase stays consistent. For instance, you renamed PIT_EXECUTION_MODE as >EXECUTION_SCOPE but did you notice that we use the term "execution mode" elsewhere? We have a PitExecutionMode enum, a setExecutionMode method, etc. It's important to keep the same terminology throughout the project otherwise it can become very confusing. In other words, if we decide to rename a constant we may have to rename classes, fields or methods as well.

The next time I will take a deeper look, before changing such constants. 

>
>Besides, if by "key" you were refering to the value of the constant (pitExecutionMode) then I don't think we'll change it because it would likely break compatibility with existing run configurations.

This was the reason, why I didn't want to change the value of the constant in the first place but would have looked odd besides the others.

Reply:
Under that respect I agree with @echebbi : let's avoid having too many changed lines in a single PR and focus on the actual additions/modifications